images/build/cross/default/Dockerfile

```diff
@@ -28,7 +28,8 @@ ENV KUBE_DYNAMIC_CROSSPLATFORMS \
   armhf \
   arm64 \
   s390x \
-  ppc64el
+  ppc64el \
+  i386
```

Comment:
Tiny nit: can we keep this list sorted?

Reply:
Yeah, it wasn't sorted before. Now it is :+1: